### PR TITLE
Add og image for five adventure page and update lfx term 03 dates

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR 
- adds og image for [adventures-of-five-and-friends](https://layer5.io/community/adventures-of-five-and-friends)
- udpates lfx term 03 dates

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a534603f2a85b2091fee0afb8ac484c0ea4d951a at: https://64c164945a7e6631f107da61--layer5.netlify.app

File: static/images/meet-five.svg
Comment (leecalcote) [inline static/images/meet-five.svg:1]: REMINDER: Please use a PNG for all OpenGraph images.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: static/images/meet-five.svg
Comment (leecalcote) [reply static/images/meet-five.svg:1]: Also, please keep the OpenGraph image in the same component / section.

Comment (vishalvivekm) [reply static/images/meet-five.svg:1]: Yes @leecalcote . Actually my build is failing locally after I pulled latest from upstream.
I'll get it fixed and then SVG -> PNG , upload the png in the same location

**Review by leecalcote — APPROVED** [review]

File: static/images/meet-five.svg
Comment (leecalcote) [reply static/images/meet-five.svg:1]: Ok, a follow up PR...

**Pull request merged**